### DISCUSSION TITLE
/blog/building-a-blog-website-with-nextjs

### DISCUSSION THREAD

**Discussion by giscus** [jasongerbes/jasongerbes.com#31] [Comments]
# /blog/building-a-blog-website-with-nextjs

Learn how I replaced my five-year-old static website with a sleek, modern Next.js and Tailwind CSS replacement during a week-long break from work.

http://localhost:3000/blog/building-a-blog-website-with-nextjs

**Comment by jasongerbes** [comment]
If you found this post helpful, please consider leaving a reaction ☝️

You're welcome to share any feedback or ask questions. I look forward to seeing what you create next!

**Comment by mengxi-ream** [comment]
hi, thanks for this amazing post, can I ask how you added this comment feature to your blog?
